TreeTracker/UI/Views/Notes/NotesViewController.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  NotesViewController.swift
+//  TreeTracker
+//
+//  Created by Robert Huber on 9/1/22.
+//  Copyright © 2022 Greenstand. All rights reserved.
+//
+
+import UIKit
+
+class NotesViewController: UIViewController, AlertPresenting {
+
+    @IBOutlet private weak var notesTextField: UITextView! {
+        didSet {
+            notesTextField.layer.cornerRadius = 5
+            notesTextField.layer.borderColor = #colorLiteral(red: 0.5254901961, green: 0.7607843137, blue: 0.1960784314, alpha: 1)
+        }
+    }
+    @IBOutlet private weak var saveButton: PrimaryButton! {
+        didSet {
+            saveButton.isEnabled = true
+            saveButton.isHidden = false
+        }
+    }
+    var viewModel: NotesViewModel? {
+        didSet {
+            viewModel?.viewDelegate = self
+            title = viewModel?.title
+        }
+    }
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        notesTextField.delegate = self
+        viewModel?.recallSavedNote()
+        if notesTextField.text.isEmpty{
+            notesTextField.textColor = UIColor.lightGray
+            notesTextField.text = L10n.Notes.placeholder
+        }
+        let toolbar: UIToolbar = UIToolbar(frame: CGRect(x: 0, y: 0, width: self.view.frame.size.width, height: 30))
```

Comment:
Maybe take a look at `KeyboardDismissing.swift` and how it is used. SignIn/SignUp are examples of using it to dismiss the keybaord.

Reply:
Originally, the textview took up so much of the screen, this protocol wasn't obvious to the user because you always tapped into the text view. I made the border more visible and moved the textview a little to make tapping outside of the textview more obvious. Now we don't need the done toolbar.

Reply:
Ah I understand - that makes sense. I think your addition of the done button was the right call then - sorry about that.

Probably worth keeping the background tap too though for consistent behaviour - although I imagine the button would get used the majority of the time.